src/__experimental__/components/form-field/form-field.component.js

```diff
@@ -41,6 +41,7 @@ const FormField = ({
   mb,
   adaptiveLabelBreakpoint,
   styleOverride = {},
+  isRequired,
```

Comment:
Do we still need to maintain the `isOptional` prop as it seems a bit redundant to support it with `isRequired` as well?

https://github.com/Sage/carbon/blob/e8767203d7fef917c96a9283350e188310e59765/src/__experimental__/components/form-field/form-field.component.js#L38

Reply:
Yep, the `isRequired` adds a label `(optional)` instead of a red asterisk. We have three states

1. implicit optional (default behaviour)
1. explicit optional `isOptional`
1. explicit required `isRequired`

Reply:
Ah I see, I think there is a bug with the `optional`  label then as I never saw it when I was trying out your changes but I saw the asterisk when I added `required`

Reply:
In fact looking closer there's actually a typo in the story I was testing it in (`childofForm` => `childOfForm`), I'll raise a pr to fix it